src/lib/QtWidgets/QSystemTrayIcon.ts

```diff
@@ -8,6 +8,15 @@ import { NodeObject } from '../QtCore/QObject';
 
 export const QSystemTrayIconEvents = Object.freeze({
     ...BaseWidgetEvents,
+    activated: 'activated',
+    messageClicked: 'messageClicked',
+});
+export const QSystemTrayIconActivationReason = Object.freeze({
```

Comment:
Nice! Could you add this to index.ts as well ?

Otherwise its perfect

Reply:
Of course, my bad :)